src/tools/check_versions/Cargo.toml

```
@@ -0,0 +1,18 @@
+## Copyright (c) 2023 Intel Corporation
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+
+[package]
+name = "check_versions"
```

Could you use a dash rather than an underscore for consistency with our other tooling:

```suggestion
name = "check-versions"
```

I think you'll also need to `git mv check_versions check-versions` and commit that for `cargo` to be happy with the rename.